### ISSUE TITLE
"View external storybook" & "Read composition docs" controls are not accessible using keyboard in Windows and Mac.

### ISSUE THREAD

**Issue by msftedad** [storybookjs/storybook#29681]
### Describe the bug

Title: "View external storybook" & "Read composition docs" controls are not accessible using keyboard in Windows and Mac.

User Impact: 
Keyboard users, including those with motor disabilities or who rely on assistive technologies, will be unable to interact with these controls, impacting their ability to access important documentation and resources and therefore they will not be able to complete their desired task. 

Actual Result: 
"View external storybook" & "Read composition docs" controls are not accessible through keyboard in Windows and Mac.

Refer Attachment: MAC_View external storybook & Read composition docs controls are not accessible using keyboard

![Image](https://github.com/user-attachments/assets/fe10cbb8-4683-4333-b99c-c7ba1fe6949d)
https://github.com/user-attachments/assets/ddfa40cb-e388-4a89-87d5-8f5cb05e93a8

Expected Result: 
"View external storybook" & "Read composition docs" controls should be accessible through keyboard using tab key/up and down arrow keys in Windows and Mac.

### Reproduction link

About / Introduction - Page ⋅ Storybook (ambitious-cliff-0c8148010.2.azurestaticapps.net) 

### Reproduction steps

1. Open URL: [About / Introduction - Page ⋅ Storybook (ambitious-cliff-0c8148010.2.azurestaticapps.net)](https://ambitious-cliff-0c8148010.2.azurestaticapps.net/?path=/story/about-introduction--page) in latest edge browser. Home page will be displayed
2. Press tab key to move to the icon beside 'React Wrappers' control and activate it using enter key.
3. Verify on pressing tab keys whether the "View external storybook" & "Read composition docs" controls are accessible through keyboard or not.

### System

```bash
Test Environment: 
OS: Windows 11 Version 24H2 OS build 26100.1742
Browser: Microsoft New Edge Version129.0.2792.52(Official build) (64-bit)
Product: Horizon Framework
URL: About / Introduction - Page ⋅ Storybook (ambitious-cliff-0c8148010.2.azurestaticapps.net)

Mac Test Environment:
OS: macOS Sequoia 15.0.1
Browser: Microsoft New Edge Version130.0.2849.56 (Official build) (64-bit)
Product: Horizon Framework
URL: About / Introduction - Page ⋅ Storybook (ambitious-cliff-0c8148010.2.azurestaticapps.net)
```

**Comment by Sidnioulz** [comment]
Thank you for this report! I would have totally missed this use case in the sidebar work to come.

Composed Storybook menu actions should be added to secondary actions as part of the Sidebar rework (cc @MichaelArestad for awareness, we can keep the existing button with `aria-hidden` and only a click handler if we want to keep the existing popup UI; let me know what you prefer). By having these items accessible via a secondary actions menu, they won't conflict with the DOM structure of the tree node responsible for collapsing/expanding child entries.

See https://github.com/storybookjs/storybook/issues/31267 where this work will be tracked.
